libcxx/include/__bit/countl.h

```diff
@@ -49,9 +55,23 @@ inline _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR int __libcpp_clz(__uint128_t __x)
   //     zeros in the high 64-bits.
   return ((__x >> 64) == 0) ? (64 + __builtin_clzll(static_cast<unsigned long long>(__x)))
                             : __builtin_clzll(static_cast<unsigned long long>(__x >> 64));
+#  endif
 }
 #endif // _LIBCPP_HAS_NO_INT128
 
+#if __has_builtin(__builtin_clzg)
+
+template <class _Tp>
```

Comment:
Why is this repeating the signature instead of just changing the body? Same for the other changes.

Reply:
Done.